app/params/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Package params defines the simulation parameters in the gaia.
```

Comment:
Are these comments still relevant to the ICS repo?

Reply:
here, let's improve all this a bit